worker/src/bin/worker.rs

```diff
@@ -27,12 +27,8 @@ struct Args {
     #[clap(long)]
     prometheus: Option<url::Url>,
 
-    /// Port to use for diagnostics web server
-    port: Option<u16>,
-
-    /// Port to use for diagnostics web server (rust)
-    #[clap(long, default_value_t = 9820)]
-    new_port: u16,
+    /// Port this worker instance can be reached on by runner
+    port: u16,
```

Comment:
This should be an option, not an argument

Reply:
Done